vscode/src/projectSystem.ts

```diff
@@ -184,6 +206,12 @@ let projectLoader: any | undefined = undefined;
 export async function loadProject(
   documentUri: vscode.Uri,
 ): Promise<[string, string][]> {
+  // If the file is not even named yet, it won't be on disk and can't be in a project
+  if (documentUri.scheme === "untitled") {
```

Comment:
Not necessary for this change. Just a minor optimization to avoid some work if the file isn't backed by disk yet.